esp/logging/test/main.cpp

```diff
@@ -24,7 +24,8 @@
 //Copy the code to esdl_binding.cpp.
 
 #ifdef USE_TEST_RESP
-                ESPLOG(LogMin,"origResp: %s", origResp.str());
+//                ESPLOG(LogMin,"origResp: %s", origResp.str());
+                LOG(LegacyMsgCat_Min,"origResp: %s", origResp.str());
```

Comment:
This is a good example of a logging entry which the developer chooses to forcefully output the statement, let's add a appropriate threshold which the developer could use 
ie.
LOG(Forced_Threshold, "some important message")